etc/microshift/manifests/opentelemetry-gateway.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: opentelemetry-gateway-config
+  namespace: observability
+data:
+  # TODO: add k8s/microshift specific configuration
+  config.yaml: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+          http:
+    processors:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 75
+        spike_limit_percentage: 15
+      batch:
+        send_batch_size: 10000
+        timeout: 10s
+    exporters:
+      otlp/host:
+        endpoint: localhost:4317
+        tls:
+          insecure: true
+          insecure_skip_verify: true
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [memory_limiter, batch]
+          exporters: [otlp/host]
+        metrics:
+          receivers: [otlp]
+          processors: [memory_limiter, batch]
+          exporters: [otlp/host]
+        logs:
+          receivers: [otlp]
+          processors: [memory_limiter, batch]
+          exporters: [otlp/host]
+---
+apiVersion: v1
+kind: Pod
```

Comment:
I wonder if we should rather create a deployment e.g. it can get restarted, or scaled.

Reply:
From my point of view the collector should have the livespan of the microshift cluster. Since its a single machine, I currently assume there is no reason to scale it.